content/soda-gettingstarted/quickstart.md

```diff
@@ -6,11 +6,14 @@ weight: 20
 
 This is a quickstart guide for users and new contributors to try and get familiar with [SODA Foundation Release](https://github.com/sodafoundation/releases) and its [projects](https://github.com/sodafoundation). It will help to install and try out the SODA Releases using the binaries with minimum configurations or settings.
 
-***Note**: *At present, this document also provides the installation steps to try out older OpenSDS releases. Soon, the support for old opensds releases shall be removed.**
+{{% notice note %}}
```

Comment:
Can we remove this section and add a different note? Because, at the end of the document, we say, we do not support :)

Reply:
This is done.

Reply:
@skdwriting Please review and approve.